src/deck/guides/3.0-upgrade.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Upgrade to Kong Gateway 3.x
+content-type: reference
+---
+
+With the [release of {{site.base_gateway}} 3.0](/gateway/changelog/), paths that use regular expressions are no longer automatically detected.
```

Comment:
Here we're talking about `paths`, but which ones? It may not be super obvious that we are referring to routes' paths